src/Utilities/ErrorHandling/Error.hpp

```diff
@@ -56,6 +57,7 @@
  */
 #define CERROR(m)                                                             \
   do {                                                                        \
+    breakpoint();                                                             \
```

Comment:
What is CERROR even for?  The documentation says it's for use in constexpr contexts, but it calls multiple non-constexpr functions so that doesn't work.

Reply:
Yes, it somehow manages to compile in a constexpr context as long as it's not triggered. I don't understand this, so it ultimately should be replaced, but I haven't figured out what the right way to handle errors in a constexpr context is

Reply:
OK.  Very strange that creating a `std::ostringstream` is a problem but creating a `std::string` is not.

Reply:
I agree, I do not understand whatsoever :/